app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+  def create
+    comment = Comment.new(params.require(:comment).permit(:post_id, :text))
+    comment.author = current_user
+    return unless comment.save
+
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+    @liked = Like.exists?(author: current_user, post: @post)
+    render template: 'posts/show'
+  end
+end
```

Comment:
- [ ] Please your comment controller should have a `new` and `create` action method. Kindly add the new action method.
```ruby
 ...

  def new
    @comment = Comment.new
  end
```

- [ ] Please there should be a form in the views/comments/new.html.erb file which accepts input for creating comments.